components/cards/card.html

```diff
@@ -0,0 +1,237 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

Comment:
Suggestion - could add author and description meta tags.

Reply:
Yes, I will add the meta tags.